tests/Domain/Composer/ComposerFacadeTest.php

```diff
@@ -257,6 +257,34 @@ public function testIsValidConstraint($expected, $version): void {
     self::assertEquals($expected, $actual, 'Correctly determined validity of version constraint.');
   }
 
+  public function configList(): array {
+    return [
+      [['platform']],
+    ];
+  }
+
+  /**
+   * @dataProvider configList
```

Comment:
...and, correspondingly:

```suggestion
   * @dataProvider providerRemoveConfig
```